src/stores/api/handlers/secrets.go

```diff
@@ -28,11 +28,16 @@ func NewSecretsHandler(s storesmanager.Manager) *SecretsHandler {
 func (h *SecretsHandler) Register(r *mux.Router) {
 	r.Methods(http.MethodPost).Path("/{id}").HandlerFunc(h.set)
 	r.Methods(http.MethodGet).Path("").HandlerFunc(h.list)
+	r.Methods(http.MethodGet).Path("/deleted").HandlerFunc(h.listDeleted)
```

Comment:
please use the same way as other handlers.

Reply:
Applying same format